frontend/lib/screens/statistics_screen.dart

```diff
@@ -116,6 +117,15 @@ class StatisticsScreenState extends State<StatisticsScreen> {
             child: Scaffold(
                 // drawer: DrawerMenu(1),
                 appBar: AppBar(
+                  flexibleSpace: Container(
+                      decoration: BoxDecoration(
+                          gradient: LinearGradient(
```

Comment:
Alright, yeah I just wanted to spice it up a bit, thought it looked a bit dull otherwise. Maybe we can go through some of the different views/screens together and try some different looks?